src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -25,6 +26,12 @@ Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII<
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
+<PlatformSection supported={["react-native"]}>
+
+Screenshots are transfered from native SDKs to React Native over the Bridge. This can be a performance bottleneck on lower end devices. This happens only when capturing error event with the screenshots enabled. React Native New Architecture will improve the performance.
```

Comment:
Thank you for the comment, how does this sound?

```suggestion
When `attachScreenshot` is enabled and an error event is captured the screenshots are transferred from native SDKs to React Native over the bridge. This can negatively impact performance on lower-end devices, however, React Native New Architecture will improve the performance of the transfer as the data are not serialized.
```

Reply:
That's much better. A couple needed commas, but otherwise this looks good:

```suggestion
When `attachScreenshot` is enabled and an error event is captured, the screenshots are transferred from native SDKs to React Native over the bridge. This can negatively impact performance on lower-end devices, however, React Native New Architecture will improve the performance of the transfer, as the data are not serialized.
```

Reply:
Thanks.